cypress/integration/ui_spec.js

```diff
@@ -247,6 +248,115 @@ describe("UI", () => {
 				.find("[class^=error__]")
 				.should("not.exist");
 		});
+
+		it("should re-enable the input field after sending the message is successful", () => {
+			const testMessage = `Test message ${Date.now()}`;
+
+			const interception = interceptIndefinitely("POST", "*/**/messages");
+
+			visitHome();
+			clickOnLauncher();
+			sendMessage(testMessage);
+
+			// Validate that the text area is disabled
+			cy.get("@app")
+				.find("[class^=chat__]")
+				.should("be.visible")
+				.find("[class^=footer__]")
+				.should("be.visible")
+				.find("[class^=text__]")
+				.should("be.visible")
+				.find("textarea")
+				.should("be.visible")
+				.should("be.disabled")
+				.then(interception.sendResponse);
```

Comment:
Zou je deze regel apart kunnen zetten. Is nu een beetje verstopt waarom je tussen deze find/should reeks en de 2e hieronder opeens een verschil verwacht. 

Ik verwacht dat als je hem op een aparte regel zet en evt. iets van een comment erbij zet waarom je nu pas `sendResponse` doet het een stuk duidelijker is.

Reply:
Dat werkte niet, kan niet meer herinneren waarom.. vgm omdat het niet synchroon is

Reply:
https://docs.cypress.io/guides/core-concepts/introduction-to-cypress#Commands-Are-Asynchronous hier staat uitgelegd dat alle `cy.*` commands asynchroon lopen en je dus niet een synchronous call er tussen kan zetten

```
cy.get("@app")
...
.should("be.disabled");

interception.sendResponse();
```

Dit zou dus niet werken omdat de `sendResponse()` eerder af gaat dan de regel erboven

Reply:
Dan moet het inderdaad zo  blijven staan.